nncf/automl/agent/ddpg/ddpg.py

```diff
@@ -0,0 +1,265 @@
+import os
```

Comment:
One important note. We need headers with the proper license to be inserted in all the new files with code (not applicable to configs).

Reply:
https://github.com/openvinotoolkit/nncf/blob/1e9f253c7a17068209078a10b9688be1a5fe61bb/nncf/quantization/precision_init/adjacent_quantizers.py#L1-L12